plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -233,6 +233,15 @@ public function admin_scripts() {
 				wp_localize_script( 'woocommerce_quick-edit', 'woocommerce_quick_edit', $params );
 			}
 
+			// Product description
+			if ( in_array( $screen_id, array( 'product' ) ) ) {
+				wp_enqueue_script( 'wc-admin-product-editor', WC()->plugin_url() . '/assets/js/admin/product-editor' . $suffix . '.js', array( 'jquery' ), $version );
+
+				wp_localize_script( 'wc-admin-product-editor', 'woocommerce_admin_product_editor', array(
+					'i18n_description' => esc_js( __( 'Product Description', 'woocommerce' ) ),
```

Comment:
I think this should be a lowercase "D" to match the other field titles on the page:

```suggestion
					'i18n_description' => esc_js( __( 'Product description', 'woocommerce' ) ),
```